src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const ACTIVE = 'ACTIVE';
+const NOT_ACTIVE = 'NOT_ACTIVE';
+
+const actionForCategory = () => ({ type: NOT_ACTIVE });
+
+const reducerForCategory = (state = [], action) => {
+  switch (action.type) {
+    case ACTIVE:
```

Comment:
- [ ] In case you renamed the action type above, remember to ensure this aligns with it.